pkg/channel/http/http_channel_test.go

```diff
@@ -1,185 +1,185 @@
-	t.Run("single concurrency", func(t *testing.T) {
-		handler := testConcurrencyHandler{
-			maxCalls: 1,
-			lock:     sync.Mutex{},
-		}
-		server := httptest.NewServer(&handler)
-		c := Channel{baseAddress: server.URL, client: &fasthttp.Client{}}
-		c.ch = make(chan int, 1)
-
-		var wg sync.WaitGroup
-		wg.Add(5)
-		for i := 0; i < 5; i++ {
-			go func() {
-				request2 := &channel.InvokeRequest{
-					Payload: []byte(""),
-				}
-				c.InvokeMethod(request2)
-				wg.Done()
-			}()
-		}
-		wg.Wait()
-		assert.False(t, handler.testFailed)
-		server.Close()
-	})
-
-	t.Run("10 concurrent calls", func(t *testing.T) {
-		handler := testConcurrencyHandler{
-			maxCalls: 10,
-			lock:     sync.Mutex{},
-		}
-		server := httptest.NewServer(&handler)
-		c := Channel{baseAddress: server.URL, client: &fasthttp.Client{}}
-		c.ch = make(chan int, 1)
-
-		var wg sync.WaitGroup
-		wg.Add(20)
-		for i := 0; i < 20; i++ {
-			go func() {
-				request2 := &channel.InvokeRequest{
-					Payload: []byte(""),
-				}
-				c.InvokeMethod(request2)
-				wg.Done()
-			}()
-		}
-		wg.Wait()
-		assert.False(t, handler.testFailed)
-		server.Close()
-	})
-}
-
-func TestInvokeWithHeaders(t *testing.T) {
-	server := httptest.NewServer(&testHandlerHeaders{})
-	c := Channel{baseAddress: server.URL, client: &fasthttp.Client{}}
-	request := &channel.InvokeRequest{
-		Metadata: map[string]string{
-			"headers": "h1&__header_equals__&v1&__header_delim__&h2&__header_equals__&v2",
-		},
-	}
-	response, err := c.InvokeMethod(request)
-	assert.NoError(t, err)
-	assert.Contains(t, string(response.Data), "H1&__header_equals__&v1")
-	assert.Contains(t, string(response.Data), "H2&__header_equals__&v2")
-	server.Close()
-}
-
-func TestContentType(t *testing.T) {
-	t.Run("default application/json", func(t *testing.T) {
-		handler := &testContentTypeHandler{}
-		server := httptest.NewServer(handler)
-		c := Channel{baseAddress: server.URL, client: &fasthttp.Client{}}
-		request := &channel.InvokeRequest{}
-		c.InvokeMethod(request)
-		assert.Equal(t, "application/json", handler.ContentType)
-		server.Close()
-	})
-
-	t.Run("application/json", func(t *testing.T) {
-		handler := &testContentTypeHandler{}
-		server := httptest.NewServer(handler)
-		c := Channel{baseAddress: server.URL, client: &fasthttp.Client{}}
-		request := &channel.InvokeRequest{
-			Metadata: map[string]string{ContentType: "application/json"},
-		}
-		c.InvokeMethod(request)
-		assert.Equal(t, "application/json", handler.ContentType)
-		server.Close()
-	})
-
-	t.Run("custom", func(t *testing.T) {
-		handler := &testContentTypeHandler{}
-		server := httptest.NewServer(handler)
-		c := Channel{baseAddress: server.URL, client: &fasthttp.Client{}}
-		request := &channel.InvokeRequest{
-			Metadata: map[string]string{ContentType: "custom"},
-		}
-		c.InvokeMethod(request)
-		assert.Equal(t, "custom", handler.ContentType)
-		server.Close()
-	})
-}
+// ------------------------------------------------------------
```

Comment:
what's the change in this file? maybe carriage return?

Reply:
yeah, it is. This is mainly a preparation step to enable linting for all `_test.go` later.